server/routes/notes.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express")
+const router = express.Router()
+const notesController = require('../controllers/notes')
+
+//crud operations
+router.get('/create', notesController.create)
+router.get('/delete', notesController.delete)
+router.get('/update', notesController.update)
+router.get('/insert', notesController.insert)
+
```

Comment:
Lets update these as per REST standards
For creating user: `router.post('/notes', notesController.create)`
For listing users: `router.get('/notes', notesController.index)`
For getting a user: `router.get('/notes/:id', notesController.get)`
For updating user: `router.put('/notes', notesController.update)`
For deleting user: `router.delete('/notes/:id', notesController.delete)`

Reply:
ok

Reply:
router.post('/create', notesController.create)
router.delete('/delete/:id', notesController.delete)
router.update('/update/:id', notesController.update)
router.get('/:id', notesController.get)
router.get('/index', notesController.index)

is this ok